pkg/encoder/encoder.go

```diff
@@ -201,14 +201,23 @@ func PrintNS(ns int32) string {
 func HumanStackTrace(response *tetragon.GetEventsResponse, colorer *Colorer) string {
 	out := new(strings.Builder)
 	if ev, ok := response.Event.(*tetragon.GetEventsResponse_ProcessKprobe); ok {
-		if ev.ProcessKprobe.StackTrace != nil {
-			for _, st := range ev.ProcessKprobe.StackTrace {
+		if ev.ProcessKprobe.KernelStackTrace != nil {
+			fmt.Fprintf(out, "Kernel:\n")
+			for _, st := range ev.ProcessKprobe.KernelStackTrace {
 				colorer.Green.Fprintf(out, "   0x%x:", st.Address)
 				colorer.Blue.Fprintf(out, " %s", st.Symbol)
 				fmt.Fprintf(out, "+")
 				colorer.Yellow.Fprintf(out, "0x%x\n", st.Offset)
 			}
 		}
+		if ev.ProcessKprobe.UserStackTrace != nil {
+			fmt.Fprintf(out, "User space:\n")
+			for _, st := range ev.ProcessKprobe.UserStackTrace {
+				colorer.Green.Fprintf(out, "   0x%x:", st.Address)
+				colorer.Blue.Fprintf(out, " %s ", st.Symbol)
+				colorer.Yellow.Fprintf(out, "(%s+0x%x)\n", st.Module, st.Offset)
```

Comment:
This is a nit but I would maybe use a slightly different print format to avoid double space `" %s "` in case we don't have the symbol. Maybe something like `symbol@module+offset` idk what the usage.

Reply:
I fixed double space. Maybe keep this output? [ASAN](https://clang.llvm.org/docs/AddressSanitizer.html) prints stack traces this way. I think there are many people familiar with ASAN output. Here is some [examples](https://www.osc.edu/resources/getting_started/howto/howto_use_address_sanitizer).

P.S. ASAN has double space problem :smile: . For me it's OK if you prefer another output format.

Reply:
reproducing ASAN output is a very nice idea and a good argument!